docs/source/rbac/upgrade.md

```diff
@@ -1,31 +1,33 @@
-# Upgrading JupyterHub with RBAC framework
+The RBAC framework new in JupyterHub 2.0. enables for the different degrees of permissions to be granted to tokens via scopes tied to roles. This, no doubt eliminates the distinction between OAuth and API tokens as was been used in previous versions - (see {ref}`oauth-vs-api-tokens-target` to learn more). As a result, the different database setup in the previous JupyterHub versions are merged into one and as such, all tokens that already exist and were created prior to the upgrade must be replaced because they are no longer compatible with the updated database.
 
-RBAC framework requires different database setup than any previous JupyterHub versions due to eliminating the distinction between OAuth and API tokens (see {ref}`oauth-vs-api-tokens-target` for more details). This requires merging the previously two different database tables into one. By doing so, all existing tokens created before the upgrade no longer comply with the new database version and must be replaced.
+Here is how it works- During the database upgrade, all existing tokens are deleted and the tokens loaded via the `jupyterhub_config.py` file are recreated with updated structure. But following the upgrade, any manually created or saved tokens must be manually reissued because they are not automatically created.
 
-This is achieved by the Hub deleting all existing tokens during the database upgrade and recreating the tokens loaded via the `jupyterhub_config.py` file with updated structure. However, any manually issued or stored tokens are not recreated automatically and must be manually re-issued after the upgrade.
-
-No other database records are affected.
+**Please note that upgrading JupyterHub does not affect any database record**.
 
 (rbac-upgrade-steps-target)=
 
 ## Upgrade steps
 
-1. All running **servers must be stopped** before proceeding with the upgrade.
-2. To upgrade the Hub, follow the [Upgrading JupyterHub](../admin/upgrading.rst) instructions.
-   ```{attention}
-   We advise against defining any new roles in the `jupyterhub.config.py` file right after the upgrade is completed and JupyterHub restarted for the first time. This preserves the 'current' state of the Hub. You can define and assign new roles on any other following startup.
-   ```
-3. After restarting the Hub **re-issue all tokens that were previously issued manually** (i.e., not through the `jupyterhub_config.py` file).
+Let's take a look at the steps to follow when upgrading JupyterHub with RBAC framework.
+
+Step1. Stop **all running servers** before proceeding with the upgrade.
```

Comment:
Starting lines with `1.` etc. is part of markdown. Let's not change that here, which will mean we don't get lists in the rendered docs.